server/dev-server.js

```diff
@@ -0,0 +1,22 @@
+async function go() {
+  const {execa} = await import('execa')
+  if (process.env.NODE_ENV === 'production') {
+    await import('../index.js')
+  } else {
+    const command =
+      'tsx watch --clear-screen=false --ignore "app/**" --ignore "build/**" --ignore "node_modules/**" --inspect ./index.js'
+    execa(command, {
+      stdio: ['ignore', 'inherit', 'inherit'],
+      shell: true,
+      env: {
+        ...process.env,
+        FORCE_COLOR: true,
+        MOCKS: true,
```

Comment:
Let's make it possible to override these values if necessary

```suggestion
        FORCE_COLOR: true,
        MOCKS: true,
        ...process.env,
```